addons/binding/org.openhab.binding.gardena/src/main/java/org/openhab/binding/gardena/internal/GardenaSmartImpl.java

```
@@ -242,12 +244,17 @@ private Devices loadDevices(Location location) throws GardenaException {
                     property.setAbility(ability);
 
                     // special conversion for pump, convert on/off to boolean
-                    if (device.getCategory().equals(DEVICE_CATEGORY_PUMP)
-                            && property.getName().equals(ABILITY_PUMP_ON_OFF)) {
-                        property.setValue(
-                                new PropertyValue(String.valueOf("on".equalsIgnoreCase(property.getValueAsString()))));
+                    if (device.getCategory().equals(DEVICE_CATEGORY_PUMP)) {
+                        if (property.getName().equals(PROPERTY_PUMP_ON_OFF)) {
+                            property.setValue(new PropertyValue(
+                                    String.valueOf("on".equalsIgnoreCase(property.getValueAsString()))));
+                        } else if (property.getName().equals(PROPERTY_MANUAL_WATERING_TIMER)) {
+                            Integer duration = getIntegerValue(property.getValueAsString());
+                            if (duration != null) {
```

Should the invalid duration be silently ignored here?